dataset/non_huggingface_datasets.py

```diff
@@ -3,7 +3,9 @@
 import os
 import json
 import random
-from urllib.request import urlretrieve
+import gdown
```

Comment:
Is gdown a new dependency? If so, you need to change the requirements.txt as well